ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
> Alternatively, if we just care about writing to an output file and reading it with python, another option would be to just remove the \r from the testcase.

The ```\r``` is there specifically to ensure that you can write binary data. I should probably make sure writing a raw ```\00``` works as well.